src/main/java/be/syntra/devshop/DevshopBack/factories/ShopOrderFactory.java

```diff
@@ -0,0 +1,40 @@
+package be.syntra.devshop.DevshopBack.factories;
+
+import be.syntra.devshop.DevshopBack.entities.OrderContent;
+import be.syntra.devshop.DevshopBack.entities.ShopOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+@Component
+public class ShopOrderFactory {
+    private final OrderContentFactory orderContentFactory;
+
+    @Autowired
+    public ShopOrderFactory(OrderContentFactory orderContentFactory) {
+        this.orderContentFactory = orderContentFactory;
+    }
+
+    public ShopOrder of(
```

Comment:
mind the spacing!

Reply:
yes,looks like i removed all args and left it like that.....